src/pages/Search/SearchAdvancedFiltersPage/SearchFiltersInPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SearchFiltersChatsSelector from '@components/Search/SearchFiltersChatsSelector';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as SearchActions from '@userActions/Search';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SearchFiltersInPage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const [isScreenTransitionEnd, setIsScreenTransitionEnd] = useState(false);
+
+    const [searchAdvancedFiltersForm] = useOnyx(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM);
+    const handleScreenTransitionEnd = () => {
+        setIsScreenTransitionEnd(true);
+    };
+
+    return (
+        <ScreenWrapper
+            testID={SearchFiltersInPage.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldShowOfflineIndicatorInWideScreen
+            offlineIndicatorStyle={styles.mtAuto}
+            onEntryTransitionEnd={handleScreenTransitionEnd}
+            shouldEnableMaxHeight
+        >
+            <HeaderWithBackButton title={translate('common.in')} />
```

Comment:
Please add another prop here with a specific goBack like this:
```
                onBackButtonPress={() => {
                    Navigation.goBack(ROUTES.SEARCH_ADVANCED_FILTERS);
                }}
```.

Also if you can take a look at the rest of pages and add it to any that are missing. I think I forgot about this in some other filters as well

Reply:
All filters checked and fixed where needed 👌